valhalla/skadi/sample.h

```diff
@@ -31,6 +38,12 @@ class sample {
    */
   template <class coord_t> double get(const coord_t& coord);
 
+  /**
+   * Get a single sample from a remote source
+   * @param coord the single posting at which to sample the datasource
+   */
+  template <class coord_t> double get_from_remote(const coord_t& coord);
```

Comment:
could you move it to the private/protected section? Looks like there's no need to reveal this method